core/cap-0042.md

```diff
@@ -254,6 +254,8 @@ Systems that try to compute transaction fees that are close to market rate will
 
 Users that just bid what they're willing to spend on fees, will not be effected.
 
+As transaction sets are transmitted over the network, the overlay network will have to be updated to support the new format.
```

Comment:
Will the xdr require other changes to overlay messages not captured the cap? or is this paragraphb just referring to the fact that the overlay messages won't be backwards compatible?

Reply:
It does require changes to `StellarMessage`.